resources/assets/js/app/views/mainLayout.js

```diff
@@ -13,9 +14,33 @@ var mainLayoutView = Marionette.LayoutView.extend({
         breadCrumbs: '#breadcrumbs',
         modalWindow: '#modal-container'
     },
+    onRender: function () {
+        if(config.externalHeader){
+            var getHeader = function() {
```

Comment:
Please move external collection header to separate method of the view. For example: initExternalView()
Then in render() will be just: 
if(config.externalHeader){
   this.initExternalView()
}